spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/imports/ImportDefinitions.groovy

```diff
@@ -0,0 +1,16 @@
+package org.springframework.cloud.contract.verifier.builder.imports
+
+/**
+ * @author Olga Maciaszek-Sharma
+ */
+
+class ImportDefinitions {
```

Comment:
`@since`, `@CompileStatic`, `@PackageScope` - I'm not sure about package scopes since people might want to reuse these classes if they want their own java tests